xml/System.Reflection.PortableExecutable/PdbChecksumDebugDirectoryData.xml

```diff
@@ -41,7 +41,7 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Checksum algorithm name.</summary>
```

Comment:
```suggestion
        <summary>Gets the checksum algorithm name.</summary>
```
But is this a "checksum algorithm name" or a "checksum based on the algorithm name"?

Reply:
The name of crypto hash algorithm used to calculate the checksum. E.g. "SHA256".